examples/controlplane/controlplane.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   nodeDrainTimeout: ${NODE_DRAIN_TIMEOUT:-"0s"}
   replicas: 3
-  version: v1.17.0
+  version: v1.21.0
```

Comment:
This is `v1.21.1` in m3-dev-env if the intention is to keep them in sync.

Reply:
I went ahead and changed v1.21.0 to v1.21.1 everywhere, that is also in e2e tests.